courses/machine_learning/deepdive2/feature_engineering/solutions/2_bqml_adv_feat_eng.ipynb

```diff
@@ -0,0 +1,2308 @@
+{
```

Comment:
<p>Delete all TODO cells. If you're worried about having to do the work later when making the student labs, you can still copy an paste from the github version history :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
All TODO cells deleted.